### DISCUSSION TITLE
Expand upon the updates to virtual environments policy

### DISCUSSION THREAD

**Discussion by JonDouglas** [actions/virtual-environments#3173] [Ideas]
From an outside perspective, I might look at the cadence and type of updates that the virtual environment takes on, so I may read the following README section:

https://github.com/actions/virtual-environments#updates-to-virtual-environments

However, one of the most important pieces of information to me might be the policy surrounding what will actually be updated to the virtual environment. 

Although the `latest` tag process may be [long and complex](https://github.com/actions/virtual-environments/issues/1816), it would be especially helpful to know the policy for each of the various types of tools, OS versions, and more so that somebody can understand at a glance what is likely going to be updated & the timeframe it may take.

For example, if there was a section or wiki that described the cadence of taking on new `Ubuntu` versions and it specifying that the policy is only LTS releases meaning even versions, that would really help. Another thought for tooling would be the same policies around SDKs and such. In a similar example above, various versions of .NET Core 3.0.X are removed due to .NET EOL policy. 

A wiki or table of some sort and a link to each of the LTS or EOL policies that the virtual environments follows would be extremely helpful!

**Comment by catthehacker** [comment]
It was just merged: https://github.com/actions/virtual-environments/pull/3127

**Reply by JonDouglas** [reply]
What great timing!

**Comment by AlenaSviridenko** [comment]
Hey!
Yeah, that idea was flying around for some time, we received several feedbacks that it is not clear how we choose what to deprecate, when we change the defaults and so on.
Minor note: that policy is what we will follow from now on, some tools' presence can mismatch this doc description, because these tools were added some time ago, and not it can be too dangerous to change them.
